tracer/src/Datadog.Trace/Debugger/Configurations/ConfigurationUpdater.cs

```diff
@@ -105,9 +116,14 @@ bool IsEnvAndVersionMatch(ProbeDefinition probe)
             }
         }
 
-        private void HandleProbesChanges(ProbeConfigurationComparer comparer)
+        private void HandleAddedProbesChanges(ProbeConfigurationComparer comparer)
```

Comment:
nit: consider passing in just `comparer.AddedDefinitions` (or possibly just inlining this entire method?)

Reply:
I prefer not.
Soon we will support `update` probe configurations, and we will need to use comparer